docs/content/guides/columns/react-hot-column.md

```diff
@@ -144,7 +144,7 @@ In this example, the custom editor component is created with an external depende
 
 ::: example #example6 :react-advanced --tab preview
 ```jsx
-import { useEffect } from 'react';
+import React, { useEffect } from 'react';
```

Comment:
It should work probably.

```suggestion
import { useEffect, useRef } from 'react';
```

Reply:
hey, we need this `React` import here, cause in few next lines we use `React.createRef()` which can not be replaced with just `useRef(null)` in this example currently

Reply:
Sure, thanks for explanation, I've missed that.